.github/workflows/go.yml

```diff
@@ -132,6 +132,30 @@ jobs:
       - name: Check out code into the Go module directory
         uses: actions/checkout@v2
 
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.0.1
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.0.0
+        with:
+          node_image: kindest/node:v1.18.15
+
+      # don't run `ct install` if charts hasn't been changed
+      - name: Run chart-testing (install)
+        run: |
+          kubectl create ns vela-system
+          kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/provider/kind/deploy.yaml
```

Comment:
why need to apply this file?

Reply:
FYI the code is not the final state now.

I want to check the `PHASE` and `HEALTH` of `applicatoin` to be `running` and `healthy` in test pod.
In my local environment the `HEALTH` of application stuck in `checkHealth`, this line is just having a try. If it's no necessary, I will  remove it.